PyInstaller/loader/rthooks/pyi_rth_kivy.py

```diff
@@ -0,0 +1,10 @@
+from os.path import join
+from os import environ
+import sys
+
+root = join(sys._MEIPASS, 'kivy_install')
+sys.path += [join(root, '_libs')]
+
+environ['KIVY_DATA_DIR'] = join(root, 'data')
```

Comment:
Sure I can change this. But just out of curiosity: Why do like it more this way?